mina-daemon/values.yaml

```diff
@@ -145,6 +145,11 @@ service:
   labels: {}
   loadBalancerClass:
   publishNotReadyAddresses: true
+  graphql:
+    type: ClusterIP
+    loadBalancerClass:
+    annotations: {}
+    publishNotReadyAddresses: false
```

Comment:
Oh I see now why it will be right, because we define the value tree. But it is confusing to me. Please refactor.

Reply:
Since we are creating multiple services(see templates) I think value tree should look like this:
```
service:
  daemon: {}
  graphql: {}
```
So graphql does not need refactoring.
Moving current `service` values is a breaking change that shouldn't be part of this pr.